include/common.php

```diff
@@ -1829,13 +1829,11 @@ function get_labels_xml_from_query_results($qry)
     return $xml;
 }
 
-function generate_web_api_key()
+function generate_password($length)
```

Comment:
Should we just be using `random_bytes` here? We already require the random compat library in PHP < 7, and PHP >= 7 provides it out of the box.

See: https://github.com/Kitware/CDash/blob/master/public/login.php#L50